x-pack/plugins/cases/public/components/user_action_tree/index.tsx

```diff
@@ -656,15 +672,12 @@ export const UserActionTree = React.memo(
           return prevManageMarkdownEditIds;
         });
 
-        if (
-          commentRefs.current &&
-          commentRefs.current[draftComment.commentId] &&
-          commentRefs.current[draftComment.commentId].editor?.textarea &&
-          commentRefs.current[draftComment.commentId].editor?.toolbar
```

Comment:
`toolbar` is not part of the `editor` object. Nevertheless, `openLensModal` check for its existence.